package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "node-api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
```

Comment:
It's good practice to add a 'start' script in your package.json. This is typically the entry point for your application when it's deployed. [medium]